domains/trist-tran.is-cool.dev.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "../schemas/domain.schema.json",
+
+	"description": "Learning deploy code to VPS",
+
+	"domain": "is-cool.dev",
+	"subdomain": "trist-tran",
+
+	"owner": {
+	  "repo": "https://github.com/Deanout/aws_video.git",
+	  "email": "hello@example.com"
+	},
+
+	"record": {
+	  "A": ["157.245.128.111"],
+	  "NS": ["ns1.digitalocean.com.", "ns2.digitalocean.com.", "ns3.digitalocean.com."]
```

Comment:
Choose either the A record or NS records.

Reply:
I choose NS records. Thanks. It's the first time I learn about it